cypress/integration/Switch.spec.ts

```diff
@@ -0,0 +1,69 @@
+import * as h from '../helpers';
+
+const getSwitch = () => {
+  return cy.get(`[type="checkbox"]`);
+};
+
+describe('Non-disabled Switch', () => {
+  before(() => {
+    h.stories.visit();
+  });
+  ['Default', 'Alert', 'Error'].forEach(story => {
+    context(`Given the '${story}' story is rendered`, () => {
+      beforeEach(() => {
+        h.stories.load('Components|Inputs/Switch/React/Top Label', story);
+      });
+
+      it('should not have any axe errors', () => {
+        cy.checkA11y();
+      });
+
+      context('when the tab key is pressed', () => {
+        it('should be the focused item on the page', () => {
+          cy.tab();
+          getSwitch().should('have.focus');
+        });
+      });
+
+      context('when the switch is clicked', () => {
+        it('should be toggled', () => {
+          getSwitch()
```

Comment:
I've been putting the actions of the `context` in a `beforeEach` of that context. This keeps the "when {something}" out of the spec itself. It allows for additional tests. So the spec is only an assertion of what we expect an outcome to be.

This block would look like:
```ts
      context('when the switch is clicked', () => {
        beforeEach(() => {
          getSwitch().click();
        });

        it('should be toggled', () => {
          getSwitch().should('be.checked');
        });
      });
```